nodes.go

```diff
@@ -225,6 +226,96 @@ func (n *Node) GetMetadata() (string, error) {
 	return md.String(), nil
 }
 
+type updateMetadata struct {
+	Name        *string `json:"name,omitempty"`
+	Description *string `json:"description,omitempty"`
+}
+
+// Updates metadata fields name or description of Node n. If a given field is nil,
+// it will not be updated.
+func (n *Node) UpdateMetadata(name *string, description *string) (*http.Response, error) {
```

Comment:
Can you return the Node that Amazon Drive returns please?  My policy is always to return stuff it if comes back in the API.  The caller can always ignore it...